### ISSUE TITLE
Support pg gem version 1.0

### ISSUE THREAD

**Issue by ribose-jeffreylau** [riboseinc/activeuuid#7]
For some reason, Travis is complaining about `gem "pg"` not being included and whatnot:

```
Add `gem 'pg'` to your Gemfile (and ensure its version is at the minimum required by ActiveRecord). (Gem::LoadError)
```

For example: https://travis-ci.org/riboseinc/activeuuid/jobs/338812989

I tried adding `require "pg"` (d2e4517b57b66) in different places (4d4daaef3e2a) but it would still complain about the same thing.  As such, I've put the `env: DB=postgres` line under `allow_failures:` in `.travis.yml` (81fde57d7686bb) just to [let Travis turn green](https://travis-ci.org/riboseinc/activeuuid/builds/338802750?utm_source=github_status&utm_medium=notification).

Any idea on how to approach this problem?

cc: @skalee

**Comment by skalee** [comment]
@ribose-jeffreylau Looks like some breaking changes in a recently released 1.0 version.

**Comment by skalee** [comment]
Downgrading helped. Almost. Builds for Postgres + Rails 4.0, and Postgres + Rails 4.1 are still failing, but the reason is different. See: https://travis-ci.org/riboseinc/activeuuid/jobs/339261632.
